.github/workflows/retryConformTests.yml

```diff
@@ -0,0 +1,42 @@
+name: Storage Retry conformance tests against service emulator
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: /home/runner/work/google-cloud-dotnet/google-cloud-dotnet/apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/
+
+    services:
+      emulator:
+        image: gcr.io/cloud-devrel-public-resources/storage-testbench:latest
+        ports:
+          - 9000:9000
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      # Install .NET Core 3.1 for testing
+      - name: Setup .NET Core 3.1
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 3.1.x
+
+      # Install .NET 6 for building
+      - name: Setup .NET 6.0
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 6.0.100
+
+      - run: dotnet test Google.Cloud.Storage.V1.RetryConformanceTests.csproj
+        env:
```

Comment:
It may be working but I'd expect the environment variable to be set prior to running the tests.

Reply:
This is not a script, it's a configuration file, this part here configures a `run` action, basically, it says: "run this command with this env var set, and continue on error".

BTW, the `continue-on-error` I think does nothing here, as you are not trying to run a script that has several steps where if one step fails, other steps may continue. You are running a single command (`dotnet test`) and if that fails there's nothing to continue with.

Reply:
Right. Thanks Amanda.